streaming/vision/convert/coco.py

```diff
@@ -200,12 +200,12 @@ def main(args: Namespace) -> None:
         args (Namespace): Command line arguments.
     """
     fields = {
-        'img': 'bytes',
-        'img_id': 'bytes',
-        'htot': 'bytes',
-        'wtot': 'bytes',
-        'bbox_sizes': 'bytes',
-        'bbox_labels': 'bytes'
+        'img': 'jpeg',
+        'img_id': 'int',
+        'htot': 'int',
+        'wtot': 'int',
+        'bbox_sizes': 'pkl',
```

Comment:
sounds like we may want to add "ndarray:(shape with 0 or 1 wildcards):dtype" in the future?

Reply:
Can you please elaborate on this ?

Reply:
just speculating on a potential future type of encoding